clarity.d.ts

```diff
@@ -35,6 +35,9 @@ interface IConfig {
   // Setting to enable debug features (e.g. console.log statements)
   debug?: boolean;
 
+  // Setting to enable consistency verifications between real DOM and shadow DOM
+  ensureConsistency?: boolean;
```

Comment:
>ensureConsistency?: boolean; [](start = 2, length = 28)

validateConsistency? By default we will try to ensure consistency but I believe you are trying here to validate that we were indeed consistent? Also a quick comment on potential cost would be nice for folks to understand the associated cost of the feature.